src/app/admin/product-form/product-form.component.html

```diff
@@ -1 +1,37 @@
-<div>Product Form Model</div>
\ No newline at end of file
+<ul ngbNav #nav="ngbNav" class="header">
+  <li ngbNavItem>
+    <div class="nav-link">
+      Add Product
+    </div>
+    <ng-template ngbNavContent>
+      <form #addProduct="ngForm" (ngSubmit)="onAdd(addProduct.value)">
+        <label for="pname">Product Name</label>
+        <input type="text" class="form-control" id="pname" name="productname" placeholder="Product name.." required
+          ngModel autofocus>
+        <div class="items">
+          <label for="price">Price(₹)</label>
+          <input type="number" id="price" name="price" placeholder="Price.." class="form-control" ngModel required>
+          <label class="label-2" for="discount">Discount(%)</label>
+          <input type="number" id="discount" name="discount" placeholder="Discount.." class="form-control" ngModel
+            required>
+        </div>
+        <label for="description">Description</label>
+        <textarea type="text" id="description" name="description" placeholder="Description.." class="form-control"
+          ngModel required></textarea>
+        <div class="items">
+          <label for="color">Color</label>
+          <input class="color-selector" type="color" id="color" name="color" class="form-control"
+            ngModel required>
+          <label class="label-2" for="size">Size</label>
+          <input type="number" id="size" name="size" placeholder="Size.." class="form-control" ngModel required>
```

Comment:
should be dropdown, u can get the values using /sizes api. and also this field should be displayed based on the categories